package.json

```diff
@@ -108,7 +110,8 @@
     "commands": "./dist/commands",
     "plugins": [
       "@oclif/plugin-help",
-      "@oclif/plugin-plugins"
+      "@oclif/plugin-plugins",
+      "@oclif/plugin-autocomplete"
```

Comment:
do we need to install @oclif/plugin-autocomplete ?

Reply:
yes

Reply:
ah, for some reason I didn't see that the dep is already where.